### DISCUSSION TITLE
Robot suddenly stops then restarts

### DISCUSSION THREAD

**Discussion by dansity** [Hypfer/Valetudo#1775] [Q&A / Support]
Hi,
I have installed Valetudo on my Roborock S1 and I started to having problem. If there is no solution I unfortunately have to go back to xiaomi fw.
Robot does the job then suddenly stops. Logging stops, web interface down. Robot spins the lidar but not moving, vacuum motor not spinning. After around 30sec it continues cleaning. Map is gone, robot thinks charger point is where it is at the moment. Does this 10 times once each minute then it is not doing it for the rest time but since the charger point is relocated it no longer homes.

This is from valetudo.log, seems like it just restarts during cleaning: https://pastebin.com/3bkHJ9Bb

Any idea what to do? Are there other logs to check?

**Comment by Hypfer** [comment]
> If there is no solution I unfortunately have to go back to xiaomi fw.

Okay. A full refund has been issued. Don't let the door hit you on the way out

**Comment by dansity** [comment]
Dude.. I came here to ask a question with good will. Your unhelpful cynical comment does not help. I was clear with my problem, I provided logs. If you don't have an answer please just leave the question open maybe in the future someone will have an answer.
Mark my question open again and please don't answer anymore.
